app/views/users/phone_setup/spam_protection.html.erb

```diff
@@ -20,12 +20,22 @@
       input.closest('form').submit();
     }
   <% end %>
-  <div
-    class="g-recaptcha"
-    data-sitekey="<%= IdentityConfig.store.recaptcha_site_key_v2 %>"
-    data-callback="onCaptchaResponse"
-  ></div>
-  <script src="https://www.google.com/recaptcha/api.js" async></script>
+  <% if IdentityConfig.store.recaptcha_site_key_v2.present? %>
+    <div
+      class="g-recaptcha"
+      data-sitekey="<%= IdentityConfig.store.recaptcha_site_key_v2 %>"
+      data-callback="onCaptchaResponse"
+    ></div>
+    <script src="https://www.google.com/recaptcha/api.js" async></script>
+  <% elsif IdentityConfig.store.phone_recaptcha_mock_validator %>
+    <%= f.input :recaptcha_mock_score, as: :hidden, input_html: { value: 1.0 } %>
+    <%= render ButtonComponent.new(id: :submit, big: true, wide: true, class: 'display-block margin-y-5') do %>
```

Comment:
Fun bug here revealed in the new feature specs, and a TIL!

A form object reference in JavaScript apparently creates properties for each of its inputs by ID, so having a button with ID `submit` overrides the form's default submit function that [we're expecting to be able to call above](https://github.com/18F/identity-idp/blob/737d05f80815c4212cb63415b77d65d0a2270b3e/app/views/users/phone_setup/spam_protection.html.erb#L20).

![image](https://user-images.githubusercontent.com/1779930/220660815-60a2a873-5437-4c09-a2fe-1158a943ff49.png)

```
Uncaught TypeError: input.closest(...).submit is not a function
```

I'll change to a unique ID that won't conflict with default form properties.

Reply:
Updated in 3c0b586f5.